### PR TITLE
NEXTEUROPA-13955: Add hook_requirement on communities features

### PR DESCRIPTION
**Pull request by gervasek**
Prevent modules communities and nexteuropa_communities to be enabled
simultaneous
NEXTEUROPA-13955

### CODE REVIEW

**Comment by drupol** [comment]
Does adding a `hook_requirements()` is enough to prevent the concurrent installations ?

**Comment by gervasek** [comment]
This is the solution used on the platform (proposed by @pfrenssen ): https://github.com/ec-europa/platform-dev/blame/master/profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.install
And it work:
```
$ drush en nexteuropa_communities
Module nexteuropa_communities doesn't meet the requirements to be enabled.
This module is not compatible with Communities. Please disable that module first.
https://github.com/ec-europa/platform-dev/blame/master/profiles/common/modules/features
```
So I thinks it's good, but we can ask Pieter for more validation.

**Comment by drupol** [comment]
It's ok I just verified: 

>  If a requirement has a severity of REQUIREMENT_ERROR, install.php will abort or at least the module will not install. 

Link: https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_requirements/7.x

**Comment by drupol** [comment]
:+1:

**Comment by drupol** [comment]
Is there a ticket for this ?

**Comment by gervasek** [comment]
Yes, in commit comment: NEXTEUROPA-13955
But this pull request is for a merge on release/communities, the review should be done by platform community team. 
After, when it will merged on release/communitites, we will open task on you bord for a merge on master branch.

**Comment by pfrenssen** [comment]
This approach looks good to me indeed.

File: profiles/common/modules/features/communities/communities.install [hunk 1]
```diff
@@ -5,6 +5,23 @@
  */
 
 /**
+ * Implements hook_requirements().
+ */
+function communities_requirements() {
+  $requirements = [];
+  if (module_exists('nexteuropa_communitites')) {
```
Comment (gervasek) [inline profiles/common/modules/features/communities/communities.install]: Well spotted !! Thanks !!

**Pull request merged**